squape/squishserver.py

```diff
@@ -133,20 +145,25 @@ def addAppPath(self, path: str) -> None:
         Args:
             path (str): the AUT path to register
         """
-        log(f"[Squishserver {self.host}:{self.port}] " f"Registering AUT path: {path}")
-        self._config_squishserver("addAppPath", [path])
+        path_resolved = PureWindowsPath(path).as_posix().strip()
```

Comment:
why PureWindowsPath and not Path?

Reply:
Because Path.as_posix() produces a string with double back-slashes ('\\') on Linux, and PureWindowsPath.as_posix() properly converts this string to contain only forward slashes ('/')